makefiles/arch/avr8.inc.mk

```diff
@@ -40,6 +40,6 @@ OPTIONAL_CFLAGS_BLACKLIST += -Wformat-truncation
 OPTIONAL_CFLAGS_BLACKLIST += -gz
 
 # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105523
-ifneq ($(findstring 12.,$(shell $(CC) --version 2>/dev/null)),)
+ifeq ($(findstring 12.,$(shell $(CC) --version 2>/dev/null)),)
```

Comment:
Huh but this does what it's supposed to do, no?
If `gcc --version` contains 12, the resulting string is not equal the empty string, so the condition is executed.

Granted, this will break again with GCC 13, but hopefully by then that bug is fixed.

Reply:
It may cause breakage if something like `5.12.3` is used. Maybe that triggered the PR?